### ISSUE TITLE
How can not Run the umlet extensions?

### ISSUE THREAD

**Issue by BojunZhong** [umlet/umlet#722]
I am a vscode extension programmer, I want to learn the umlet extensions. Well I just want to Run that, generally, run vscode extensions can be used as F5. It was showed:
![1685948798619(1)](https://github.com/umlet/umlet/assets/102637083/86920366-8b05-4826-bfa4-acd47f912ffd)

**Comment by afdia** [comment]
I dont think it is caused bei Umlet (we don't require a browser path), maybe another extension is the problem.
You could try it on a new installation of vscode without other extensions

**Comment by BojunZhong** [comment]
Thanks for replying, well, if you want to run the umlet extensions, what is the process will you do?

**Comment by afdia** [comment]
As mentioned in https://marketplace.visualstudio.com/items?itemName=TheUMLetTeam.umlet

> For a new diagram, create an empty text file with a ".uxf"-extension!

Or just open an existing uxf diagram in vscode and it will use the Umlet plugin
